modules/core/shared/src/main/scala/io/circe/ACursor.scala

```diff
@@ -222,6 +222,15 @@ abstract class ACursor(private val lastCursor: HCursor, private val lastOp: Curs
    */
   def downField(k: String): ACursor
 
+  /**
+   * Navigates down into the specified fields of a JSON object sequentially.
+   *
+   * @param k  The first field name to navigate into.
+   * @param ks Additional field names to navigate into, in sequence.
+   * @return An updated cursor focused on the final field in the sequence, or a failed cursor if any field is not found.
+   */
+  def downFields(k: String, ks: String*): ACursor
```

Comment:
To address the mima warning that showed up in CI, we need to make sure any new methods added to traits have a default implementation  I think the implementation used by `HCursor`  could probably just be moved here and solve the problem. 

```suggestion
  def downFields(k: String, ks: String*): ACursor = {
    ks.foldLeft(this.downField(k))(_.downField(_))
  }
```

Reply:
Ok. I'll follow this suggestion. 👌